core/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -694,7 +702,8 @@ class QualificationAppInfo(
         origPlanInfos, origPlanInfosSummary.map(_.stageSum).flatten,
         perSqlStageSummary.map(_.stageSum).flatten, estimatedInfo, perSqlInfos,
         unSupportedExecs, unSupportedExprs, clusterTags, allClusterTagsMap, mlFunctions,
-        mlTotalStageDuration, unsupportedExecExprsMap)
+        mlTotalStageDuration, unsupportedExecExprsMap,
+        clusterInfo, eventLogInfo.map(_.eventLog.toString))
```

Comment:
`eventLogInfo.map(_.eventLog.toString)` could we construct the clusterSummary with the information of app/app-name/and eventlogpath? we don't need to pass the eventlog as argument, right?

Reply:
Wrapped it in a case class `ClusterSummary`